es_utils.py

```diff
@@ -136,7 +136,7 @@ def get_payload_wscroll(index, query, max_count=-1):
 
 def get_template(index=''):
   data = {'index':index, 'api': '/_template', 'prefix': True}
-  return ssl_urlopen(CMSSDT_ES_QUERY,json.dumps(data))
+  return ssl_urlopen(CMSSDT_ES_QUERY,json.dumps(data).encode())
```

Comment:
@iarspider , I think we need to `decode` the return value of `ssl_urlopen` e.g. https://github.com/cms-sw/cms-bot/pull/2057/files#diff-403eb1824bda3fe3ae2b63977f99ec9b6b2911b348f16801dc8733e019dea46bL47 
For me the following fails for python3
```
python3 -c 'from es_utils import find_indexes;print(find_indexes("cmssdt-ib-dataset-*"))'
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/tmp/cmsbuild/del/cms-bot/es_utils.py", line 143, in find_indexes
    for line in get_indexes(index).split("\n"):
TypeError: a bytes-like object is required, not 'str'
```